pages/content/posts/localizing-your-website.md

```diff
@@ -0,0 +1,29 @@
+
+Going back to our previous example of missing alt. text attributes, let’s say that you translate a portion of your website, but don’t translate everything.  Perhaps you’ve translated the content but not the interface, or vice versa, or perhaps you’ve only translated part of the content.  Now the user is viewing content on your page in two different languages, but expects to only see one.  Machine translation software is now out of the question, as it is likely to end up picking up already-translated words and attempt to “translate” them again, or be unable to pick up what the original language is.  “What happened?” the user will think. “Is the website broken?  I can’t understand half of this…I guess I should search elsewhere.”
+
+And that’s the real kicker.  Even if the user understands the original language, they would still think that something is wrong, which would provide a negative user experience and give your competitors an opportunity to gain your lost consumers.  However, if a user is requesting an alternate language, chances are high they can’t understand the original language in the first place!  By leaving your product partially translated (or realistically, untranslated at all), you are essentially telling the users that speak other languages: “we really don’t care about you.”
+
+Another benefit of localizing your website is that it can improve search engine optimization (SEO).  Search engines are more likely to rank websites that are localized for different languages higher in search results, as they are seen as providing a better user experience for users who speak those languages, the same way that websites with accessibility considerations are ranked higher.  By localizing a website, you can increase your visibility in search results and have a greater chance at gaining more organic traffic.
+
+Of course, it is also important to _maintain_ your localizations, rather than performing them once and calling it good.  As you update your website and add or modify content, it's important to ensure that the localized versions are kept up-to-date and accurately reflect the latest content.  This can help prevent users from coming across outdated or inaccurate information, which leads to a negative user experience and potentially drives users away from the website.
```

Comment:
```suggestion
Of course, it is also important to _maintain_ your localizations, rather than performing them once and forgetting about them.  As you update your website by adding or modifying content, it's important to ensure the localized versions are also kept up-to-date and always accurately reflect the latest content.  This helps prevent users from encountering outdated or inaccurate information, which leads to a negative user experience and potentially drives users away from the website.
```